docs/github/continuous-integration.md

```diff
@@ -0,0 +1,284 @@
+
+## Pull Requests
+A pull request (PR) is a way to propose changes to a code repository. It is a formal request to the repository owner(s) to merge your changes into their codebase, often the “main” branch.
+
+PRs are typically created by cloning or forking the repository, making changes to the code, and then submitting a request to the repository owner to merge your changes. The repository owner can then review your changes, request changes – if necessary, and decide whether to merge them into their codebase.
+
+Here are some of the benefits of using PRs:
+
+- **Collaboration**: PRs allow developers to collaborate on code and share ideas.
+- **Code review**: PRs allow other developers to review your changes and provide feedback.
+- **Testing**: PRs can be used to test changes before they are merged into the main codebase.
+- **Documentation**: PRs can be used to document changes to the code.
+- **History**: PRs provide a history of changes to the code.
+
+## Creating or switching branches
+
+1. Go to your Repository > https://github.com/cagov/[repsoitory_name]
+2. Click on the Branch Dropdown
+    1. On the main page of your repository, you'll see a dropdown menu on the top left, displaying the current branch (usually “main”). Click on this dropdown.
+3. Type or click on an Branch Name
+    1. In the dropdown that appears, there’s a textbox at the top, type the name for the branch you want to switch to or create.
+    2. When creating a new branch, choose a descriptive and relevant name. e.g. `project_staging_model`
+    3. **If you’re just switching branches you are done after this step!**
+4. Choose a base for the New Branch
+    1. Usually this is “main” as you want to work off of the most up-to-date version of code, but you can also choose another branch to base your new one off of if you want to add to someone else’s work in progress without making edits directly to their branch and causing a merge conflict! 😬
+5. Click "Create branch `<branch_name>` from `<base_branch_name>`" (usually your base branch will be “main”)
+
+Now, you have successfully created a new branch in your GitHub repository and will be automatically switched to this branch. You can make changes on this new branch without affecting the main branch.
+
+If you're working with git locally, you can create and switch branches as well as check your current git branch using the following git commands:
+
+- **Create and switch to a branch:** `git switch -c <branch_name>`
+    - **This also works, but is the older way of doing it:** `git checkout -b <branch_name>`
+- **Switch to an existing branch:** `git switch <branch_name>`
+    - If this doesn’t work it’s likely that you have to pull down the remote branch you want to work with first using `git fetch`
+    - Then you can run git switch `<branch_name`> again
+
+🛎️Note: replace `<branch_name>` with the actual name of your new branch e.g. `writing_project_docs`.
```

Comment:
Our mkdocs config has a way to create note boxes using `!!! note` (you can find examples of it elsewhere in the docs)

Reply:
here's an example (after the External users code block): https://github.com/cagov/data-infrastructure/blob/76329617bcfc3910a8477bf0f62d09251f1dc8e4/docs/code/local-setup.md#configure-dbt

and here's what it looks like: https://cagov.github.io/data-infrastructure/code/local-setup/#configure-dbt

Reply:
Thanks! I've updated the doc to use note boxes.